README.md

```diff
@@ -79,3 +79,16 @@ $ docker run --rm -e LS_ACCESS_TOKEN -e OTEL_EXPORTER_OTLP_TRACES_ENDPOINT -e TO
 ```
 
 When building with Docker, you need to re-run both steps for any code *or* config changes. If you run into errors while building, please open [an issue](https://github.com/lightstep/telemetry-generator).
+
+### Non-development Workflow: Run a published image
+
+If you just want to run (vs build or develop), you can run the most recent image published to this repository's container registry. 
+
+For defaults, see `Dockerfile`.
+
+```shell
+$ export LS_ACCESS_TOKEN=your token
+# can override to any other OTLP endpoint
+$ export OTEL_EXPORTER_OTLP_TRACES_ENDPOINT=ingest.lightstep.com:443
+$ docker run -e LS_ACCESS_TOKEN --rm lightstep/telemetry-generator:latest
```

Comment:
q: The run command will need to point to the ghcr registry, right?

Reply:
yes, correct! will update shortly.